.github/pull_request_template.md

```diff
@@ -1,23 +1,35 @@
+
 This code review checklist is intended to serve as a starting point for the author and reviewer, although it may not be appropriate for all types of changes (e.g. fixing a spelling typo in documentation).  For more in-depth discussion of how we think about code review, please see [Code Review Guidelines](../blob/main/docs/CODE_REVIEW_GUIDELINES.md).
 
 # Author
-<!-- NOTE: Do not modify these when initially opening the pull request.  This is a checklist template that you tick off AFTER the pull request is created. -->
-- [ ] Self-review: Did you review your own code in GitHub's web interface? _Code often looks different when reviewing the diff in a browser, making it easier to spot potential bugs._
-- [ ] Automated tests: Did you add appropriate automated tests for any code changes?
-- [ ] Manual tests: Did you update the [manual tests](../blob/main/docs/testing/manual_testing) as appropriate? _While we aim for automated testing of the application, some aspects require manual testing. If you had to manually test something during development of this pull request, write those steps down._
-- [ ] Code coverage: Did you check the code coverage report for the automated tests?  _While we are not looking for perfect coverage, the tool can point out potential cases that have been missed._ Code coverage can be generated with: `./gradlew check` for Kotlin modules and `./gradlew connectedCheck -PIS_ANDROID_INSTRUMENTATION_TEST_COVERAGE_ENABLED=true` for Android modules.
-- [ ] Documentation: Did you update documentation as appropriate? (e.g [README.md](../blob/main/README.md), etc.)
-- [ ] Run the app: Did you run the app and try the changes?
-- [ ] Screenshots: Did you provide before and after UI screenshots in the description of this pull request?  _This is only applicable for changes that modify the UI._
-- [ ] Rebase and squash: Did you pull in the latest changes from the main branch and squash your commits before assigning a reviewer? _Having your code up to date and squashed will make it easier for others to review. Use best judgement when squashing commits, as some changes (such as refactoring) might be easier to review as a separate commit._
 
+- [ ] **Self-review** your own code in GitHub's web interface[^selfReview]
+- [ ] Add appropriate **automated tests** if needed
+- [ ] Update the [**manual tests**](../blob/main/docs/testing/manual_testing)[^manTest] as appropriate
+- [ ] Check the **code coverage**[^coverage] report for the automated tests
+- [ ] Update **documentation** as appropriate? (e.g [README.md](../blob/main/README.md), etc.)
+- [ ] **Run the app** and try the changes?
+- [ ] Pull in the latest changes from the **main** branch and **squash** your commits before assigning a reviewer[^pullSquash]
+
+> **Note**
+> It is good practice to provide before and after UI  **screenshots** in the description of this PR This is only applicable for changes that modify the UI.
```

Comment:
I'm not sure whether I have an opinion about this being a checkbox versus being just a note.

We haven't had a ton of UI changes in PRs recently, so as a checkbox it gets under utilized. However we do expect more UI changes in the future.

If we keep it as a note, the text could be updated to read: "For changes that modify the UI, it is good practice to provide before and after **screenshots** in the description of this PR."

Reply:
I think it depends on how do we want to communicate it.  
If we leave it as a note it is more of a suggestion, but we really don't care if you do it or not.
If we change it to checkbox it is perceived more as a requirement (no mater if mandatory, or optional)
what do you think? @HonzaR @ccjernigan